dialogue-annotations/build.gradle

```diff
@@ -12,6 +12,12 @@ dependencies {
     implementation 'com.google.code.findbugs:jsr305'
     implementation 'com.palantir.safe-logging:preconditions'
     implementation 'com.palantir.safe-logging:safe-logging'
+    implementation 'com.google.guava:guava'
```

Comment:
Dunno why this needs to be here? I didn't add any guava things...

Reply:
huh, I'm not sure.

Reply:
huh, I'm not sure.